libs/common/src/lib/ai-models/summarization/summarization.component.html

```diff
@@ -30,7 +28,7 @@
           id="summary">
           @for (option of options; track option.value) {
             <pa-radio [value]="option.value">
-              {{ option | learningOption: 'summary' }}
+              {{ option.name }}
```

Comment:
should it be `'kb.settings.learning-options.summary-' + option.name` as before ? or `option.name` is good enough ?

Reply:
the backend provides a proper name, no need for translation